packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -25,6 +27,16 @@ function MyFeedHeading({
     return <FeedHeading>My feed</FeedHeading>;
   }
 
+  const { trackEvent } = useContext(AnalyticsContext);
+
+  const onClick = () => {
+    trackEvent({
```

Comment:
You are missing the event name here.
```suggestion
    trackEvent({
    event_name: 'click manage tags'
```

Preferably convert this into the enum that holds Impression though.

Reply:
Chris is right there can't be an event without event_name
Also, target_id and target_type are not required here. I left a comment for Sab on the taxonomy.

Reply:
I have missed this completely. I have now updated our `AnalyticsEvent` type to have the property `event_name` required as it was previously optional to avoid missing this one.